### ISSUE TITLE
Front game page

### ISSUE THREAD

**Issue by EhJay22** [OurTreatApp/OurTreat-iOS#125]
Bold the companies name.

**Comment by EhJay22** [comment]
Waiting on V12 to roll out to test this.

**Comment by EhJay22** [comment]
@shreyansjain it wasn't moved up, but we've now created a separate issue so I'll close this.
